src/app/work-item/work-item.service.ts

```diff
@@ -2,7 +2,7 @@ import { Link } from '../models/link';
 import { Injectable, Component } from '@angular/core';
 import { Headers, Http } from '@angular/http';
 import { cloneDeep } from 'lodash';
-
+import { Observable }     from 'rxjs';
```

Comment:
Why did you import rxjs like this?
due to a change in rxjs, we import it differently like the Promise in the line below